python/__init__.py

```diff
@@ -0,0 +1,13 @@
+#  Copyright 2021 Rikai Authors
```

Comment:
Do we need this level ? 🤣

Reply:
I just have intellij automatically add it to all .py files lol